build-system/tasks/helpers.js

```diff
@@ -143,12 +143,10 @@ async function compileBentoRuntime(options) {
  * @return {!Promise}
  */
 async function compileAllJs(options) {
+  log('Compiling JS with', cyan('esbuild'), 'and', cyan('babel') + '...');
+
   const {minify} = options;
-  if (minify) {
-    log('Minifying multi-pass JS with', cyan('closure-compiler') + '...');
-  } else {
-    log('Compiling JS with', cyan('esbuild'), 'and', cyan('babel') + '...');
-  }
```

Comment:
I think the distinction between minified and non-minified ("compiled", for lack of a better word) ought to remain in the logs. Perhaps change the first word of the sentence to `minify ? 'Minifying' : 'Compiling'`?

Reply:
Is there a distinction between the two with esbuild?

Reply:
Sure. I'll put that distinction back in via `Compiling minified` vs. `Compiling unminified`, which I think reads clearer.
WDYT of removing the details around esbuild/babel? I think those might be too much detail for this log (why are we excluding the other dozen tools...where do we draw the line..)

Reply:
> Sure. I'll put that distinction back in via `Compiling minified` vs. `Compiling unminified`, which I think reads clearer.

SGTM. 👍 

> WDYT of removing the details around esbuild/babel? I think those might be too much detail for this log (why are we excluding the other dozen tools...where do we draw the line..)

Fair point. The logging was originally added back when there was a substantial difference between minified (only closure) and unminified (only babel). It's less accurate / useful today given all the recent tooling changes.